<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <author>
SPDX-FileCopyrightText: none
SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-KDE-Accepted-GPL
 </author>
 <class>MultipleMarkerDialog_UI</class>
 <widget class="QDialog" name="MultipleMarkerDialog_UI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>309</width>
    <height>218</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Marker</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" colspan="2">
    <widget class="QLabel" name="clip_filesize_2">
     <property name="text">
      <string>Start Position</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2" colspan="3">
    <widget class="TimecodeDisplay" name="m_in"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="clip_filesize_3">
     <property name="text">
      <string>Comment</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2" colspan="3">
    <widget class="QLineEdit" name="marker_comment"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="clip_filesize_4">
     <property name="text">
      <string>Interval</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2" colspan="3">
    <widget class="TimecodeDisplay" name="interval"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="clip_filesize_5">
     <property name="text">
      <string>Count</string>
     </property>
    </widget>
   </item>
   <item row="3" column="2" colspan="3">
    <widget class="QSpinBox" name="occurrences">
     <property name="minimum">
      <number>2</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_category">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="4" column="2" colspan="3">
    <widget class="MarkerCategoryChooser" name="marker_category">
     <property name="allowAll">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="3">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="1" colspan="2">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>18</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="4">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TimecodeDisplay</class>
   <extends>QWidget</extends>
   <header>widgets/timecodedisplay.h</header>
  </customwidget>
  <customwidget>
   <class>MarkerCategoryChooser</class>
   <extends>QComboBox</extends>
   <header>widgets/markercategorychooser.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MultipleMarkerDialog_UI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MultipleMarkerDialog_UI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
